src/main/java/net/minecraftforge/client/model/ModelLoader.java

```diff
@@ -214,6 +214,17 @@ protected ResourceLocation getModelLocation(ResourceLocation model)
     {
         return new ResourceLocation(model.getResourceDomain(), model.getResourcePath() + ".json");
     }
+    
+    @Override
+    public ModelBlockDefinition getModelBlockDefinition(ResourceLocation location)
```

Comment:
This was added to expose ModelBakery.getModelBlockDefinition() as it is protected, to make lives easier should anyone need it.